### PR TITLE
fix cnl issue

### PR DESCRIPTION
**Pull request by peggy-gov-bc-ca**
# Description

This PR includes the following proposed change(s):

- Fix could not locate for real estate and bank info
## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Tested locally

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/DynamicsAdapter/DynamicsAdapter.Web/Keys.cs [hunk 1]
```diff
@@ -15,4 +15,9 @@ public static class Keys
         public static readonly string REDIS_KEY_PREFIX = "dyna_";
         public static readonly string DATA_PROVIDER_KEY = "ProviderList";
     }
+
+    public static class Constants
+    {
+        public static readonly string CouldNotLocate = "Could Not Locate";
+    }
 }
```
Comment (rantifamilusi) [inline app/DynamicsAdapter/DynamicsAdapter.Web/Keys.cs:23]: new line required

File: app/DynamicsAdapter/DynamicsAdapter.Web/Keys.cs [hunk 2]
```diff
@@ -15,4 +15,9 @@ public static class Keys
         public static readonly string REDIS_KEY_PREFIX = "dyna_";
         public static readonly string DATA_PROVIDER_KEY = "ProviderList";
     }
+
+    public static class Constants
+    {
+        public static readonly string CouldNotLocate = "Could Not Locate";
+    }
```
Comment (rantifamilusi) [inline app/DynamicsAdapter/DynamicsAdapter.Web/Keys.cs:22]: Can we use  `const` instead of readonly CouldNotLocate

**Review by rantifamilusi — APPROVED** [review]

**Review by rantifamilusi — COMMENTED** [review]
Looks good but some comments

File: app/DynamicsAdapter/DynamicsAdapter.Web/Keys.cs (hunk 2 above)
Comment (peggy-gov-bc-ca) [reply app/DynamicsAdapter/DynamicsAdapter.Web/Keys.cs:22]: sure

**Review by rantifamilusi — APPROVED** [review]

**Pull request merged**